docs/guides/administrator/gcp-setup.md

```diff
@@ -0,0 +1,160 @@
+gcloud projects add-iam-policy-binding ${GCP_PROJECT_ID} "--member=serviceAccount:${GCP_SVC_ACC}" \
+    --role=roles/editor
+gcloud projects add-iam-policy-binding ${GCP_PROJECT_ID} "--member=serviceAccount:${GCP_SVC_ACC}" \
+    --role=roles/iam.securityAdmin
+```
+
+Repeat the command from above, listing the security roles assigned to the service account in the current project:
+
+```text
+$ gcloud projects get-iam-policy ${GCP_PROJECT_ID}  \
+    --flatten="bindings[].members" \
+    --format='table(bindings.role)' \
+    --filter="bindings.members:${GCP_SVC_ACC}"
+ROLE
+roles/compute.networkAdmin
+roles/container.admin
+roles/editor
+roles/iam.securityAdmin
+```
+
+## Step 5: enable GCP APIs
+
+Cloud DNS API:
+
+```text
+$ gcloud services enable dns.googleapis.com "--project=${GCP_PROJECT_ID}"
+Operation "operations/<snip>" finished successfully.
+```
+
+Compute Engine API:
+
+```text
+$ gcloud services enable compute.googleapis.com "--project=${GCP_PROJECT_ID}"
+Operation "operations/<snip>" finished successfully.
+```
+
+Kubernetes Engine API:
+
+```text
+$ gcloud services enable container.googleapis.com "--project=${GCP_PROJECT_ID}"
+Operation "operations/<snip>" finished successfully.
+```
+
+Cloud SQL Admin API:
+
+```text
+$ gcloud services enable sqladmin.googleapis.com "--project=${GCP_PROJECT_ID}"
+Operation "operations/<snip>" finished successfully.
+```
+
+Service Networking API:
+
+```text
+$ gcloud services enable servicenetworking.googleapis.com "--project=${GCP_PROJECT_ID}"
+Operation "operations/<snip>" finished successfully.
+```
+
+Cloud Resource Manager API:
+
+```text
+$ gcloud services enable cloudresourcemanager.googleapis.com "--project=${GCP_PROJECT_ID}"
+Operation "operations/<snip>" finished successfully.
+```
+
+## Step 6: create a key and credentials file for the service account
+
+Issue the following command to generate a private key for the service account, and to store it together with credential metadata in a JSON file (in your current working directory):
+
+```text
+$ gcloud iam service-accounts keys create \
+    --iam-account "${GCP_SVC_ACC}" \
+    "${GCP_PROJECT_ID}-svc-acc.json"
+created key [<snip>] of type [json] as [<project>-svc-acc.json] for [<project>-svc-acc@<project>.iam.gserviceaccount.com]
+```
+
+Lock the file permissions down:
+
+```text
+chmod 600 "${GCP_PROJECT_ID}-svc-acc.json"
+```
+
+Now, move this credential file wherever you need it to be and remember to always treat it securely: for a production workflow, you may want to submit it to a system designed for storing secrets, and then wipe it from your local machine.
+
+## Further steps
+
+For creating an Opstrace cluster using this project and service account, all you need to do is to set the environment variable `GOOGLE_APPLICATION_CREDENTIALS` (to point to the credentials file created in step 7) before invoking `opstrace create gcp ...`. For further information, please have a look at the [CLI reference documentation](./cli.md).
```

Comment:
```suggestion
For creating an Opstrace cluster using this project and service account, all you need to do is to set the environment variable `GOOGLE_APPLICATION_CREDENTIALS` (to point to the credentials file created in step 7) before invoking `opstrace create gcp ...`. For further information, please have a look at the [CLI reference documentation](../../references/cli.md).
```

I think this is the right path from here.

Reply:
Oh right!

Reply:
Commit pushed.
Also added a line break between sentences.